pkg/kubelet/config/flags.go

```diff
@@ -94,7 +94,7 @@ func (s *ContainerRuntimeOptions) AddFlags(fs *pflag.FlagSet) {
 
 	// Network plugin settings for Docker.
 	fs.StringVar(&s.NetworkPluginName, "network-plugin", s.NetworkPluginName, fmt.Sprintf("<Warning: Alpha feature> The name of the network plugin to be invoked for various events in kubelet/pod lifecycle. %s", dockerOnlyWarning))
-	fs.StringVar(&s.CNIConfDir, "cni-conf-dir", s.CNIConfDir, fmt.Sprintf("<Warning: Alpha feature> The full path of the directory in which to search for CNI config files. %s", dockerOnlyWarning))
+	fs.StringVar(&s.CNIConfDir, "cni-conf-dir", s.CNIConfDir, fmt.Sprintf("<Warning: Alpha feature> A comma-separated list of full paths of directories in which to search for CNI config files. %s", dockerOnlyWarning))
```

Comment:
I expected this to be a `StringArray` or `StringSlice` if we want to make it multi-value

Reply:
does this need a description of how precedence is handled if a plugin with the same name exists in multiple dirs?

Reply:
as written, this changes the behavior if a `--cni-conf-dir` path containing a comma was provided. `fs.StringArray` would preserve the current behavior, and would specify multiple dirs with multiple parameters (`--cni-conf-dir=dir1 --cni-conf-dir=dir2`)